etc/ci/update_nightlies.sh

```diff
@@ -62,14 +62,15 @@ MASTER_COMMIT="$(git rev-parse HEAD)"
 
 git checkout -b gh-pages upstream/gh-pages || exit 1
 
-git rm -rf nightly || true
-rm -rf nightly || true
+OLD_NIGHTLIES="$(find nightly -type f -mtime +1)"
```

Comment:
You might want to add a note that the removal will fail if any of the file names contain spaces.  Also, add `2>/dev/null` to the `find` command, so we don't get spew to stderr if `nightly` doesn't exist.